### ISSUE TITLE
Uncaught TypeError: Argument 1 passed to Prompt_Rest_Api::init() 

### ISSUE THREAD

**Issue by Dannyholt** [beaverbuilder/assistant#143]
**Reported by a user from help desk**

I've had issues with a WP error. It was saying it was the Postmatic plugin:

`PHP Fatal error: Uncaught TypeError: Argument 1 passed to Prompt_Rest_Api::init() must be an instance of WP_REST_Server, string given, called in /www/wp-includes/class-wp-hook.php on line 286 and defined in /www/wp-content/plugins/postmatic/core/rest-api.php:26`

**Stack trace:**
```php
#0 /www/wp-includes/class-wp-hook.php(286): Prompt_Rest_Api::init('') 
#1 /www/wp-includes/class-wp-hook.php(310): WP_Hook->apply_filters(NULL, Array) 
#2 /www/wp-includes/plugin.php(465): WP_Hook->do_action(Array) 
#3 /www/wp-content/plugins/assistant/classes/class-fl-assistant-heartbeat.php(20): do_action('rest_api_init') 
#4 /www/wp-includes/class-wp-hook.php(288): FL_Assistant_Heartbeat::received(NULL, Array) 
#5 /www/wp-includes/plugin.php(208): WP_Hook->apply_filters(NULL, Array) 
#6 /www/wp-admin/includes/ajax-actions.php(3156): apply_filters('heartbeat_recei...', Array, Array, 'edit-thirstylin...') 
#7 /www/wp-includes/class-wp-hook.php(286): wp_ajax_heartbeat('') 
#8 /www/wp-includes/class-wp-hook.php(310): WP_Hook->apply_filters('', Array) 
#9 /www/wp-inc in /www/wp-content/plugins/postmatic/core/rest-api.php on line 26 
```

So, I contacted the plugin developer, and they came back with the following: 
 
> It looks the error is originating in the Assistant plugin. It is firing the rest_api_init action without the required argument: https://developer.wordpress.org/reference/hooks/rest_api_init/ 
If you give them that information with your error stack trace hopefully they'll fix it for you!

**Comment by brentjett** [comment]
Is this something @fastlinemedia or @phredeye want to look at? Sounds like REST stuff.

**Comment by fastlinemedia** [comment]
Fixed in 6513a73
